frontend-web/webclient/app/Applications/Overview.tsx

```diff
@@ -170,16 +175,16 @@ interface TagGridProps {
     favoriteStatus: FavoriteStatus;
     onFavorite: (app: ApplicationSummaryWithFavorite) => void;
     linkToRun?: boolean;
+    refreshId: number;
 }
 
 const TagGrid: React.FunctionComponent<TagGridProps> = (
-    {tag, columns, rows, tagBanList = [], favoriteStatus, onFavorite, linkToRun}: TagGridProps
+    {tag, columns, rows, tagBanList = [], favoriteStatus, onFavorite, linkToRun, refreshId}: TagGridProps
 ) => {
     const showFavorites = tag == SPECIAL_FAVORITE_TAG;
     const [appResp, fetchApplications] = useCloudAPI<UCloud.Page<ApplicationSummaryWithFavorite>>(
         {noop: true},
         emptyPage,
-        {cacheKey: "TagGrid" + tag, cacheTtlMs: TEN_MINUTES_IN_MILLIS}
```

Comment:
Removing this line stops caching from working, forcing a full reload on every Apps page access.